chronobit/chains.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "../chains.schema.json",
+  "name": "ChronoBit|Enterprise",
+  "chains": [
+    {
+      "name": "sentinel",
+      "address": "sentvalcons136f8p02t8903trmdt8ekrs3l4r0ursas7wj8vx",
```

Comment:
I'll merge but this is usually sentvaloper1.. address rather than sentvalcons. Might need changing

Reply:
Let me double check and confirm

Reply:
Can you update it or do I have to do another pull request? I have the valoper address.

Reply:
I'm not at a computer right now so would be quicker to submit another PR